ext/MatrixAlgebraKitCUDAExt/yacusolver.jl

```diff
@@ -306,6 +306,73 @@ function Xgesvdr!(A::StridedCuMatrix{T},
     return S, U, Vᴴ
 end
 
+# Wrapper for general eigensolver 
+for (celty, elty) in ((:ComplexF32, :Float32), (:ComplexF64, :Float64), (:ComplexF32, :ComplexF32), (:ComplexF64, :ComplexF64))
+    @eval begin
+        function Xgeev!(A::StridedCuMatrix{$elty}, D::StridedCuVector{$celty}, V::StridedCuMatrix{$celty})
+            require_one_based_indexing(A, V, D)
+            chkstride1(A, V, D)
+            n = checksquare(A)
+            # TODO GPU appropriate version
+            #chkfinite(A) # balancing routines don't support NaNs and Infs
+            n == length(D) || throw(DimensionMismatch("length mismatch between A and D"))
+            if length(V) == 0
+                jobvr = 'N'
+            elseif length(V) == n*n
+                jobvr = 'V'
+            else
+                throw(DimensionMismatch("size of VR must match size of A"))
```

Comment:
I guess we just call it `V` instead of `VR` in our function signatures and doc strings.